conf.yaml

```
@@ -127,8 +127,8 @@ contents:
             prefix:     en/welcome-to-elastic
             current:    *stackcurrent
             index:      welcome-to-elastic/index.asciidoc
-            branches:   [ 8.4 ]
-            live:       [ 8.4 ]
+            branches:   [ 8.5 ]
```

We're going to start having multiple versions of this book:
```suggestion
            branches:   [ 8.5, 8.4 ]
```